source/patterns/@aws-solutions-constructs/aws-events-rule-sns/README.md

```diff
@@ -86,8 +87,8 @@ _Parameters_
 
 Out of the box implementation of the Construct without any override will set the following defaults:
 
-### Amazon CloudWatch Events Rule
-* Grant least privilege permissions to CloudWatch Events to publish to the SNS Topic.
+### Amazon EventBridge Rule
```

Comment:
This is fine, not mandatory - it's in the README.md for the new name that we really must do this.